go.mod

```diff
@@ -30,3 +30,7 @@ require (
 	k8s.io/klog/v2 v2.3.0
 	sigs.k8s.io/kube-storage-version-migrator v0.0.3
 )
+
+replace github.com/openshift/library-go => github.com/Danil-Grigorev/library-go v0.0.0-20200917075037-909099bedd91
+
+replace github.com/openshift/api => github.com/Danil-Grigorev/api v0.0.0-20200914130008-956aacd684e2
```

Comment:
shouldn't these be pointing to the openshift/library-go repo after some commit has merged?

Reply:
Yep, was previously using my branch for testing.